src/ShapeCrawler/ShapeCollection/ISlideShapes.cs

```diff
@@ -47,6 +47,11 @@ public interface ISlideShapes : IShapes
     /// </summary>
     void AddRoundedRectangle(int x, int y, int width, int height);
 
+    /// <summary>
+    ///     Adds a new Rectangle: Top Corners Rounded. 
+    /// </summary>
+    void AddRounded2SameRectangle(int x, int y, int width, int height);
```

Comment:
Please rename this on `AddTopCornersRoundedRectangle()`.

The idea is to do naming on the _possibility_ of how it looks like in PowerPoint.

Reply:
Will do. I originally had it that way, but then saw that `GeometryType` exposed the `Round2SameRectangle` concept into the ShapeCrawler namespace.

@ashahabov What do you think of also renaming the `GeometryType` enum values to something more congruent with the UI?

Reply:
I agree, we should rename them.